src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -593,6 +593,156 @@ describe('SamDebugConfigurationProvider', async function () {
             assertEqualLaunchConfigs(actualNoDebug, expectedNoDebug)
         })
 
+        it('target=code: typescript', async function () {
+            const appDir = pathutil.normalize(
+                path.join(testutil.getProjectDir(), 'testFixtures/workspaceFolder/ts-plain-sam-app/')
+            )
+            const folder = testutil.getWorkspaceFolder(appDir)
+            const input = {
+                type: AWS_SAM_DEBUG_TYPE,
+                name: 'very good name',
```

Comment:
the names of other configs in this test file are intentionally named like `test-js-template`. Any hints that we get during debugging or when a test fails are helpful. So we should avoid funny names :)